server/services/meilisearch/connector.js

```diff
@@ -99,7 +99,13 @@ module.exports = ({ strapi, adapter, config }) => {
         adapter.addCollectionNamePrefixToId({ entryId, contentType })
       )
 
-      return await client.index(indexUid).deleteDocuments(documentsIds)
+      const task = await client.index(indexUid).deleteDocuments(documentsIds)
+
+      strapi.log.info(
```

Comment:
Can we have a test to cover those calls?

Reply:
I covered the ones that could be simply tested. The other ones are harder. Related to this issue https://github.com/meilisearch/strapi-plugin-meilisearch/issues/64